plugins/plugin-hosted/codenvy-machine-hosted/src/main/java/com/codenvy/machine/HostedMachineProviderImpl.java

```diff
@@ -210,7 +211,7 @@ protected void pullImage(CheServiceImpl service,
                                               .withBuildArgs(buildArgs),
                               progressMonitor);
 
-        } catch (ImageNotFoundException e) {
+        } catch (DockerException e) {
```

Comment:
I wonder how this behaves in case image is present but some other error occurred?